src/main/java/io/specto/hoverfly/junit/core/Hoverfly.java

```diff
@@ -178,10 +176,24 @@ public void close() {
      */
     public void importSimulation(SimulationSource simulationSource) {
         LOGGER.info("Importing simulation data to Hoverfly");
-        simulationSource.getSimulation().ifPresent(s ->
-                hoverflyResource.path(SIMULATION_PATH)
-                        .type(MediaType.APPLICATION_JSON_TYPE)
-                        .put(s));
+
+        simulationSource.getSimulation().ifPresent((Simulation s) -> {
+            try {
+
+                final byte[] jsonContent = OBJECT_MAPPER.writeValueAsBytes(s);
+                RequestBody body = RequestBody.create(JSON, jsonContent);
+
+                final Request.Builder builder = createRequestBuilderWithUrl(SIMULATION_PATH);
+                final Request request = builder.put(body).build();
+
+                final Response response = client.newCall(request).execute();
+                if (!response.isSuccessful()) throw new IOException("Unexpected code " + response);
```

Comment:
YEs this is something not clear to me, look this example from its website https://github.com/square/okhttp/wiki/Recipes#accessing-headers